shinydashboard/lantern/server.R

```diff
@@ -416,4 +475,73 @@ function(input, output, session) { #nolint
               options = list("plugins" = list("remove_button"), "create" = TRUE, "persist" = FALSE))
     }
   })
+
+  #                                          #
+  #   Display Resource and Profile Filters   #
+  #                                          #
+
+  output$profile_filter_tab <- renderUI({
```

Comment:
shouldn't this be the resource_filter_tab and the one below the profile_filter_tab?

Reply:
Yeah, no idea how I swapped these by accident! Good catch!